aws/sdk/benchmarks/s3-express/benches/concurrent_put_get.rs

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_config::BehaviorVersion;
+use aws_sdk_s3::primitives::ByteStream;
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use s3_express::{confidence_level, number_of_iterations, sample_size};
+use tokio::runtime::Runtime;
+use tokio::task;
+
+pub fn concurrent_put_get(c: &mut Criterion) {
```

Comment:
**question**: Where does this benchmark come from? What purpose is it serving (i.e. what are we measuring here)?

Reply:
It comes from the internal benchmark doc.